static/openapi/faas-spec.json

```diff
@@ -0,0 +1,670 @@
+            "in": "header",
+            "description": "The customer ID",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "functionName",
+            "in": "path",
+            "description": "The function name",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "metaOnly",
+            "in": "query",
+            "description": "If true, it specifies if want to download the meta data only but if false, the file will be downloaded",
+            "required": true,
+            "schema": {
+              "type": "boolean"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the function."
+          },
+          "404": {
+            "description": "Not found."
+          },
+          "500": {
+            "description": "Internal server error."
+          }
+        }
+      }
+    },
+    "/api/faas/v1/download/{functionName}/{version}": {
+      "get": {
+        "tags": [
+          "Function"
+        ],
+        "summary": "Download the function",
+        "description": "Download the function for the specified key function name.",
+        "operationId": "handle_mm_download",
+        "parameters": [
+          {
+            "name": "X-PhotonIQ-CustomerID",
+            "in": "header",
+            "description": "The customer ID",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "functionName",
+            "in": "path",
+            "description": "The function name",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "version",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "metaOnly",
+            "in": "query",
+            "description": "If true, it specifies if want to download the meta data only but if false, the file will be downloaded",
+            "required": true,
+            "schema": {
+              "type": "boolean"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the function."
+          },
+          "404": {
+            "description": "Not found."
+          },
+          "500": {
+            "description": "Internal server error."
+          }
+        }
+      }
+    },
+    "/api/faas/v1/function/{functionName}": {
+      "delete": {
+        "tags": [
+          "Function"
+        ],
+        "summary": "Delete All versions",
+        "description": "Delete All versions for the specified function name.",
```

Comment:
```suggestion
        "description": "Delete all versions of the specified function.",
```

Reply:
I like your edit, it's what I would have put.